test/functional/test_short_deck.py

```diff
@@ -170,3 +171,52 @@ def test_pre_flop_pot(n_players: int, small_blind: int, big_blind: int):
     assert (
         n_bet_chips == pot.total
     ), f"small and big blind have are not in pot! {n_bet_chips} == {pot.total}"
+
+
+def _play_game_helper(state, betting_round_dict, bad_seq):
```

Comment:
as a habit if you can leave a 1 liner docstring here just to describe at a glance what this does that would be awesome

Reply:
if you wanna document the parameters that's always a bonus!

Reply:
Sounds good